examples/Slider.tooltipnone.tsx

```diff
@@ -26,7 +28,14 @@ export default () => {
 
   return (
     <div style={{ width: 'min(100%, 300px)' }}>
+      <Text id={labelId} as='label' variant='leading'>
+        Choose a start date
+      </Text>
       <Slider
+        thumbProps={() => ({
+          'aria-labelledby': labelId,
+          'aria-valuetext': dateFormatter.format(currentValue.date),
```

Comment:
i feel like `aria-valuetext` should not need to be set by users. why was this needed?

Reply:
This was not needed (did not cause any a11y violation). But since the screen reader would just read out the date number (e.g. `35`), I thought of adding `aria-valuetext` with the formatted date string (e.g. something like `4 Feb`)